model-optimizer/extensions/ops/elementwise.py

```diff
@@ -88,6 +88,13 @@ class Add(Elementwise):
     op_type = 'Add'
     operation = staticmethod(lambda a, b: a + b)
 
+    def __init__(self, graph: Graph, attrs: dict):
+        attrs.update({'auto_broadcast': 'numpy'})
+        super().__init__(graph, attrs)
+
+    def backend_attrs(self):
+        return ['auto_broadcast']
+
```

Comment:
Why is this change in the `Add` class only? I expect it to be as part of base classes `Elementwise`.

Reply:
Done, this change applied to base class `Elementwise` and `backend_attrs` updated for all operations according to to their specifications. Also updated `backend_attrs` for `Broadcast` and `FakeQuantize` operations, where attribute 'auto_broadcast' was missed too.